shared/macros-bash.jinja

```diff
@@ -638,7 +638,7 @@ zypper install -y "{{{ package }}}"
 {{%- if pkg_manager == "yum" or pkg_manager == "dnf" -%}}
 if rpm -q --quiet "{{{ package }}}" ; then
 {{% if SSG_TEST_SUITE_ENV %}}
-    {{{ pkg_manager }}} remove -y --setopt protected_packages= "{{{ package }}}"
+    rpm -e --nodeps "{{{ package }}}"
```

Comment:
Won't this break the test suite on non-rpm systems?

Reply:
Yes, here we can probably use similar approach as in https://github.com/ComplianceAsCode/content/blob/master/shared/macros-bash.jinja#L608

Reply:
No, it is included in the conditional for `yum` and `dnf` only. Distros with other package managers are not affected.